stage1/init/init.go

```diff
@@ -248,6 +250,11 @@ func stage1() int {
 
 func main() {
 	flag.Parse()
+
+	if !debug {
```

Comment:
would you mind inverting the logic to make it consistent with stage0?  https://github.com/coreos/rkt/blob/master/stage0/run.go#L108